manifests/g/Genivia/ugrep/4.3.2/Genivia.ugrep.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Genivia.ugrep
+PackageVersion: 4.3.2
+PackageLocale: en-US
+Publisher: Genivia Inc.
+PublisherUrl: https://www.genivia.com
+PublisherSupportUrl: https://github.com/Genivia/ugrep/issues
+Author: Robert van Engelen
+PackageName: ugrep
+PackageUrl: https://github.com/Genivia/ugrep

```

```suggestion
PackageUrl: https://ugrep.com/
```

https://github.com/Genivia/ugrep/releases/tag/v4.3.2
> Please consider updating your links to point to ugrep.com instead of the ugrep GitHub project web site, when applicable.

---

@genivia-inc bump